src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobSignatureVerifier.java

```diff
@@ -4,33 +4,40 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.blobrouter.exceptions.DocSignatureFailureException;
+import uk.gov.hmcts.reform.blobrouter.exceptions.InvalidConfigException;
 import uk.gov.hmcts.reform.blobrouter.exceptions.InvalidZipArchiveException;
 import uk.gov.hmcts.reform.blobrouter.util.zipverification.ZipVerifiers;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
+import java.util.Base64;
 import java.util.zip.ZipInputStream;
 
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
 import static org.slf4j.LoggerFactory.getLogger;
 
 @Component
 public class BlobSignatureVerifier {
 
     private static final Logger logger = getLogger(BlobSignatureVerifier.class);
 
-    private final String publicKeyDerFilename;
+    private final String publicKeyBase64;
```

Comment:
Maybe it would be worth adding a comment or sth to contain information about the key being a der file.

Reply:
see constructor below